core/env/safe.go

```diff
@@ -108,29 +108,6 @@ func SecretDeleteBufferSizeForSafe() int {
 	return l
 }
 
-// K8sSecretDeleteBufferSizeForSafe returns the buffer size for the VSecM Safe
-// Kubernetes secret deletion queue.
-//
-// The buffer size is determined by the environment variable
-// VSECM_SAFE_K8S_SECRET_DELETE_BUFFER_SIZE.
-//
-// If the environment variable is not set, the default buffer size is 10.
-// If the environment variable is set and can be parsed as an integer,
-// it will be used as the buffer size.
-// If the environment variable is set but cannot be parsed as an integer,
-// the default buffer size is used.
-func K8sSecretDeleteBufferSizeForSafe() int {
```

Comment:
We are not using this function anymore since our k8s secrets generation logic has changed.

Right now, k8s secrets are created, updated, but not deleted.

Later, we might think about how to handle the “deletion” use case.
There are issues about that too.